### ISSUE TITLE
Serverless commands exit after a brief delay without printing any output

### ISSUE THREAD

**Issue by dougmoscrop** [serverless/serverless#9047]
When I execute any serverless command ~other than `help`~ there is a brief pause and then the process exits.

<details>
<summary><code>serverless.yml</code></summary>
I can't provide this but the issue only appears when a specific, but not only, ${file(xyz.yml)} directive is present (so there were two file imports for yml)
</details>

<details>
<summary><b><code> sls info --stage=test</code> output</b></summary>

```
```
(this space intentionally left blank)
</details>

<b>Installed version</b>

```
Framework Core: 2.28.1
Plugin: 4.4.3
SDK: 2.3.2
Components: 3.7.1
```

This appears to only happen on Windows, and 2.27 is also affected.

My debugging showed that it seems to get stuck in variable resolution, never returning from the Promise.all returned from the VariableResolver constructor.

**Comment by medikoo** [comment]
@dougmoscrop thanks for report, still just with this information I'm not able to reproduce the error. 

I've created a simple test service that does two imports on same yml file: https://github.com/medikoo/test-serverless/tree/issue-9047 and it deploys without the issue on Windows:

![Screenshot 2021-03-02 at 19 52 00](https://user-images.githubusercontent.com/122434/109699377-cd038880-7b90-11eb-9a5f-1ff19e1f7eef.png)

Can you help us getting the reproduction case (technically having `serverless.yml` with content of all file imports, but with sensitive data masked out should do the job)?

> When I execute any serverless command other than help

It's also quite surprising that you say that `help` works. New variable resolution also is used with help command and does all the variable resolution before outputting help command output (and that handling is no different in case of other commands).

**Comment by medikoo** [comment]
On Slack I learned that problem can be reproduced with:

_serverless.yml_
```yaml
custom:
  xyz: "${file(xyz.yml)}"
  baz: "${self:custom.xyz.foo}"
```

_xyz.yml_
```yaml
foo: "foo"
bar: "${self:custom.baz}"
```

Will be fixed with #9057
